src/main/java/crazypants/enderio/machine/solar/SolarPanelNetwork.java

```diff
@@ -61,21 +89,53 @@ boolean addToNetwork(TileEntitySolarPanel panel) {
     return true;
   }
 
+  void cleanupMemberlist() {
+    if (!panels.isEmpty()) {
```

Comment:
Consider reverting the direction of these if clauses.
like:
if(panels.isEmpty()) {
  return;
}

Having less layers of indentation will make this easier to read